src/main/java/MessagePrinter.java

```diff
@@ -0,0 +1,133 @@
+import tasks.Task;
+import java.util.ArrayList;
+
+public abstract class MessagePrinter {
+    // constants storing printing messages
+    private static final String FRIDAY = "  __      _     _             \n"
+            + " / _|    (_)   | |            \n"
+            + "| |_ _ __ _  __| | __ _ _   _ \n"
+            + "|  _| '__| |/ _` |/ _` | | | |\n"
+            + "| | | |  | | (_| | (_| | |_| |\n"
+            + "|_| |_|  |_|\\__,_|\\__,_|\\__, |\n"
+            + "                         __/ |\n"
+            + "                        |___/ \n";
+    private static final String DASHES = "____________________________________________________________";
+
+    // methods to print constants
+    private static void friday() {
```

Comment:
Consider renaming the methods to be verbs instead, following here: https://nus-cs2113-ay2122s1.github.io/website/se-book-adapted/chapters/codeQuality.html#use-nouns-for-things-and-verbs-for-actions